platform/ios/src/MGLAnnotationView.h

```diff
@@ -41,6 +41,19 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, assign, getter=isFlat) BOOL flat;
 
 /**
+ This property defaults to NO and becomes YES when the view is tapped/clicked on.
```

Comment:
As with the `MKAnnotationView.selected` property, the documentation for this property should mention that you shouldn’t set it directly.